dcpquery/db/models/__init__.py

```diff
@@ -94,6 +99,13 @@ class File(DCPQueryModelHelper, SQLAlchemyBase):
     def select_file(cls, file_fqid):
         return config.db_session.query(cls).filter(cls.fqid == file_fqid).one_or_none()
 
+    @classmethod
+    def select_files_for_uuid(cls, file_uuid):
+        files = config.db_session.query(cls).filter(cls.uuid == file_uuid).all()
+        if len(files) == 0:
```

Comment:
I'm a little confused by this conditional. Shouldn't there only ever be one file for one file_uuid? If there is more than one, that seems like an error as well?

This also goes for the `get` function which uses `select_files_for_uuid`. It seems like you iterate through a list of files that are associated with this uuid but it seems like that wouldn't really make sense?

Reply:
Files are specified by uuid and version (or fqid). There can be many files associated with the same uuid. The filter function will return an empty array if no matches are found.